os_net_config/utils.py

```diff
@@ -649,6 +650,21 @@ def get_vf_devname(pf_name, vfid):
     return vf_nic[0]
 
 
+def backup_map_files(backup_path):
+    src_files = [common.DPDK_MAPPING_FILE,
+                 common.SRIOV_CONFIG_FILE,
+                 sriov_config._UDEV_LEGACY_RULE_FILE
+                 ]
+    for src in src_files:
+        if os.path.exists(src):
+            src_name = os.path.basename(common.DPDK_MAPPING_FILE)
+            bkup_file = os.path.join(backup_path, src_name)
```

Comment:
Can we back this up to a time stamped name rather than a fixed name just in case the operation had to be run more than once? It would be a shame to end up overwriting an only version of the file if migration failed the first time and had to be run again. Or is this scenario unlikely to occur?
I haven't gone back to check if the calling function already puts a timestamp in the backup path, in which case ignore this comment.

Reply:
The caller is yet to be added. Its there in PR 212. I think we can create a directory with timestamp and move the files  there. If that is fine, this function can stand as it is.